src/style.css

```diff
@@ -1 +1,65 @@
-/* Styles go here */
+* {
+  font-family: "Roboto", sans-serif;
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
```

Comment:
use reseting only to that element that you need, instead of useing for * selector